OpenRA.Mods.Common/Widgets/Logic/Installation/ModContentLogic.cs

```diff
@@ -112,7 +112,7 @@ void PopulateContentList()
 				requiredWidget.IsVisible = () => p.Value.Required;
 
 				var sourceWidget = container.Get<ImageWidget>("SOURCE");
-				var sourceTitles = p.Value.Sources.Select(s => sources[s].Title).Distinct().ToList();
+				var sourceTitles = p.Value.Sources.Select(s => sources[s].Title).ToHashSet();
```

Comment:
`Distinct().ToList()` seems safer to keep the sources in the same order as listed in the file, which might have meaning.

Reply:
I changed the logic so that it wouldn't need to cast to collection type at all